distribution/docs/src/main/resources/_contents/_sources/solrcloud-contents.adoc

```diff
@@ -0,0 +1,108 @@
+ | |           |    |   |         +---------+ |
+ | |           |    |   |                     |
+ | |           |    |   | +---------+         |
+ | |           |    |   | |Solr cFF0|         |
+ | \-----------/    |   | +---------+         |
+ |                  |   |                     |
+ +------------------+   +---------------------+
+....
+
+[NOTE]
+====
+Solr Cloud is currently in Beta version.
+It is implemented but has not been tested thoroughly.
+Bugs and feature changes may affect the final release.
+====
+
+===== When to Use Solr Cloud
+
+If the ${branding} needs to be accessible for a higher than normal period with low downtime, then Solr Cloud should be used. See below for details.
+
+====== Tradeoffs
+* Scalable. Can exceed ~2 billion indexed documents.
+* Solr Admin UI can be used.
+* Has network overhead and requires additional protection to be secure.
+* Installation is more involved (requires Zookeeper)
+* Configuration and administration is more complex due to replicate, sharding, etc.
+* No way to backup currently, but will automatically recover from system failure.
+
+===== Installation/Configuration Guide
+
+The instructions on setting up Solr Cloud for ${branding} only include setup in a *NIX environment.
+
+====== Prerequisites
+
+Before the ${ddf-solr} Application can be installed:
+
+* the ${branding} Kernel must be running,
+* the ${branding} Platform Application must be installed, and
+* the ${branding} Solr Application must be installed
+* ZooKeeper 3.4.5
+* NIX environment
+* JDK 8 or greater
+
+====== Installing Zookeeper
+
+Refer to https://zookeeper.apache.org/doc/r3.1.2/zookeeperStarted.html#sc_Download for installation instructions.
+
+[NOTE]
+====
+Minimum three Zookeeper nodes required.
+Three Zookeeper nodes are needed to form a quorum.
+A three Zookeeper ensemble allows for a single server to fail and the service will still be available.
+More Zookeeper nodes can be added to achieve greater fault tolerance. The total number of nodes must always be an odd number.
+====
+
+====== Installing Solr Cloud
+
+Repeat the following procedure for each node that will be part of the Solr Cloud cluster:
+
+. Refer to https://cwiki.apache.org/confluence/display/solr/Apache+Solr+Reference+Guide for installation instructions.
+. Download jar files.
+The jars are needed to support geospatial and xpath queries and need to be installed on every "node" after the Solr Cloud installation instructions have been followed.
+
+.. http://artifacts.codice.org/service/local/repositories/releases/content/org/codice/thirdparty/jts/1.12_1/jts-1.12_1.jar
+.. http://artifacts.codice.org/service/local/artifact/maven/content?r=snapshots&g=ddf.platform.solr&a=solr-xpath&v=${ddf.version}
+
+. Copy downloaded jar files to: `<SOLR_INSTALL_DIR>/server/solr-webapp/webapp/WEB-INF/lib/`
+
+[NOTE]
+====
+Minimum two Solr server instances required with each Solr server instance having a two shard minimum.
```

Comment:
I believe the second half is saying each Solr instance should have a minimum of two shards. I broke it out into a separate sentence for clarity. Also, I changed it so we use the word "node" with Zookeeper, but for Solr Cloud we use "Solr server instance".